playwright/src/main/java/com/microsoft/playwright/impl/ElementHandleImpl.java

```diff
@@ -282,8 +283,18 @@ public byte[] screenshot(ScreenshotOptions options) {
         }
       }
     }
+    List<Locator> mask = options.mask;
+    options.mask = null;
     JsonObject params = gson().toJsonTree(options).getAsJsonObject();
+    options.mask = mask;
     params.remove("path");
+    if (mask != null) {
+      JsonArray maskArray = new JsonArray();
+      for (Locator locator : mask) {
+        maskArray.add(((LocatorImpl) locator).toProtocol());
```

Comment:
Instead of duplicating code with PageImpl, let's add a serializer for LocatorImpl.class into Serialization.java, that way it will automatically work for Locator.screenshot, ElementHandle.screenshot and Page.screenshot.

Reply:
Yes, that's what I meant, thanks!